build.gradle

```diff
@@ -213,8 +213,18 @@ project('segmentstore:storage:impl') {
 
         compile group: 'org.rocksdb', name: 'rocksdbjni', version: rocksdbjniVersion
 
+        compile project(':common')
+        compile project(':segmentstore:storage')
+        compile project(':shared:metrics')
+        testCompile project(':test:testcommon')
+        testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
+    }
+}
+
+project ('contrib:store') {
```

Comment:
Why do we need to create a `contrib` project? I think what we need to do is:

1- Either leave the common code where it is today, in `segmentstore`, or move it to `shared` if  there is any need to share code across projects. Right now, it feels like the former is more desirable.
2- Consider moving the bindings to separate repos, and if that's not possible, then we need something like a `bindings` project. One reason for having a `bindings` project is to be able to have bindings that we formally support.